src/Sankey.js

```diff
@@ -0,0 +1,201 @@
+import {sankey, sankeyLinkHorizontal} from "d3-sankey";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Path, Rect} from "d3plus-shape";
+import {dataLoad as load, Viz} from "d3plus-viz";
+
+/**
+    @class Rings
+    @extends external:Viz
+    @desc Creates a sankey visualization based on a defined set of XXXXX. [Click here](xxxxxx) for help getting started using the Sankey class.
+*/
+export default class Sankey extends Viz {
+
+  /**
+      @memberof Sankey
+      @desc Invoked when creating a new class instance, and sets any default parameters.
+      @private
+  */
+  constructor() {
+    super();
+    this._nodeId = false;
+    this._links = accessor("links");
+    this._noDataMessage = false;
```

Comment:
here's the biggest comment: override the default `mouseenter` and `mouseleave` events so that when the user interacts it also highlights the links. Take a look at the defaults [here](https://github.com/d3plus/d3plus-viz/tree/832868a117d409a6ccbd2c4a9ff85c880a747a31/src/on), you'll have to use the linkLookup in conjuction with the `hover` method.